### PR TITLE
Upversion TALM 4.18

### PR DESCRIPTION
**Pull request by sudomakeinstall2**

### CODE REVIEW

File: ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-4.17.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    workflow: optional-operators-ci-aws
+- as: security
+  optional: true
+  steps:
+    env:
+      PROJECT_NAME: cluster-group-upgrades-operator
+      SNYK_CODE_ADDITIONAL_ARGS: --severity-threshold=high
+    workflow: openshift-ci-security
+zz_generated_metadata:
+  branch: main
```
Comment (jc-rh) [inline ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-4.17.yaml:92]: release-4. 17

**Review by nishant-parekh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**